api/edgeproto/cloudlet.proto

```diff
@@ -578,6 +578,8 @@ message Cloudlet {
   string dns_label = 54 [(protogen.backend) = true];
   // Root LB FQDN which is globally unique
   string root_lb_fqdn = 55 [(protogen.backend) = true];
+  // Root LB FQDN which was set at startup and cannot be changed
```

Comment:
`set at startup` sounds like it could be referring to startup of process/service, I suggest `set when object was created`, like you have for AppInst